### PR TITLE
add docker build job in goreleaser + Dockerfile

### PR DESCRIPTION
**Pull request by Nexucis**
* Didn't get for the moment how goreleaser will login to docker.io

related to the issue #146

### CODE REVIEW

File: .goreleaser.yml [hunk 1]
```diff
@@ -30,3 +30,14 @@ checksum:
   name_template: 'checksums.txt'
 snapshot:
   name_template: "{{ .Tag }}-next"
+dockers:
+  -
+    goos: linux
+    goarch: amd64
```
Comment (slrtbtfs) [inline .goreleaser.yml:36]: I guess having images for other architectures (at least arm64) doesn't hurt.

**Comment by slrtbtfs** [comment]
> Didn't get for the moment how goreleaser will login to docker.io.

The PR shouldn't be merged until this is figured out, right?

**Comment by Nexucis** [comment]
Yeah it's safer to not merge it. To avoid a build failure during a release of the repo

File: .goreleaser.yml (hunk 1 above)
Comment (Nexucis) [reply .goreleaser.yml:36]: That would mean I have to create multiple different configuration to create and push the docker image. I'm not sure it's worth it.
We could still do it later if someone is asking. But I doubt a bit on that.

**Comment by Nexucis** [comment]
So I added a command to manually login to the docker registry.
Someone who has the right will have to fill the variable in circle-ci. ( And I don't have it and I don't know the login actually ^^)

**Comment by Nexucis** [comment]
fixes #146

**Pull request merged**